src/TerminalFighter/MainMenu/MainMenu.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <chrono>
+#include <unistd.h>
+
+#include <SDL_ttf.h>
+#include <SDL2/SDL.h>
+
+#include "../TextArea/TextArea.h"
+
+enum Options { START, QUIT };
```

Comment:
This should get moved into the class so it's private to the class. 

https://stackoverflow.com/questions/2503807/declaring-an-enum-within-a-class

you still put it in the public section though, but it doesn't pollute the global namespace.